### PR TITLE
Shims repair style

### PR DESCRIPTION
**Pull request by hyuraku**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----
repair style of bash files in `shims` for `brew style` with shellcheck `--enable=all`
https://github.com/Homebrew/brew/blob/fa3af8d24d0335641e58a4355fa9de6d42ae4982/Library/Homebrew/style.rb#L188

related #11086

### CODE REVIEW

File: Library/Homebrew/shims/linux/super/make [hunk 1]
```diff
@@ -1,26 +1,27 @@
 #!/bin/bash
 
+# shellcheck disable=SC2154,SC2250
```
Comment (MikeMcQuaid) [inline Library/Homebrew/shims/linux/super/make:3]: Can you add similar comments above this (and other `disable=`s)?

File: Library/Homebrew/shims/mac/super/apr-1-config [hunk 2]
```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
-if [[ "$HOMEBREW_CCCFG" = *a* ]]
+# shellcheck disable=SC2154
```
Comment (MikeMcQuaid) [inline Library/Homebrew/shims/mac/super/apr-1-config:3]:
```suggestion
# HOMEBREW_CCCFG and HOMEBREW_SDKROOT are set by extend/ENV/super.rb
# shellcheck disable=SC2154
```

File: Library/Homebrew/shims/mac/super/make [hunk 3]
```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-if [[ -n "$HOMEBREW_MAKE" && "$HOMEBREW_MAKE" != "make" ]]
+# shellcheck disable=SC2154
```
Comment (MikeMcQuaid) [inline Library/Homebrew/shims/mac/super/make:3]: I think the use of `HOMEBREW_MAKE` can be removed as it's undocumented and unset.

File: Library/Homebrew/shims/mac/super/pkg-config [hunk 4]
```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 
-pkg_config="$HOMEBREW_OPT/pkg-config/bin/pkg-config"
+# shellcheck disable=SC2154
```
Comment (MikeMcQuaid) [inline Library/Homebrew/shims/mac/super/pkg-config:3]:
```suggestion
# HOMEBREW_OPT and HOMEBREW_SDKROOT are set by extend/ENV/super.rb
# shellcheck disable=SC2154
```

**Review by MikeMcQuaid — COMMENTED** [review]
Nice work so far! Have made some suggestions but if you can update the rest consistently that'd be handy. I've been using e.g. `rg "HOMEBREW_OPT(\"\]\s)*="` to find definitions using `ripgrep` (hope that helps)

File: Library/Homebrew/shims/mac/super/make (hunk 3 above)
Comment (hyuraku) [reply Library/Homebrew/shims/mac/super/make:3]: >  the use of HOMEBREW_MAKE can be removed

I understood, I will open another PR to remove HOMEBREW_MAKE statement from `shims/mac/super/make`, `shims/linux/super/make` and `shims/mac/super/gmake`

File: Library/Homebrew/shims/linux/super/make (hunk 1 above)
Comment (hyuraku) [reply Library/Homebrew/shims/linux/super/make:3]: OK, I will add comments like `HOMEBREW_LIBRARY is set by bin/brew` with using ripgrep.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Nice work @hyuraku!

**Pull request merged**